build/Generator/ApiGenerator.php

```diff
@@ -61,6 +61,13 @@ public function generateOperation(ServiceDefinition $definition, string $service
         $namespace->addUse($inputClass);
         $classes = $namespace->getClasses();
         $class = $classes[\array_key_first($classes)];
+        if (null !== $version = $definition->getSignatureVersion()) {
+            $class->getMethod('getSignatureVersion')
+                ->setReturnType('string')
+                ->setVisibility(ClassType::VISIBILITY_PROTECTED)
+                ->setBody('return ' . \var_export($version, true) . ';');
+        }
```

Comment:
You can do the very same thing with service name, right?

Reply:
exact